app/serializers/external_registry_bike_v3_serializer.rb

```diff
@@ -1,8 +1,8 @@
-class ExternalBikeV3Serializer < BikeV2Serializer
+class ExternalRegistryBikeV3Serializer < BikeV2Serializer
   delegate \
```

Comment:
I don't know that you need to include this `delegate` at all?

Reply:
Is it because of the `allow_nil`?

If so, if we're using inheritance, we could specify the attributes on the `ExternalRegistryBike` base model.

Reply:
Also, I don't know why `date_stolen_string` was added in #1190 - it shouldn't have been. Sorry for not catching this earlier! It shouldn't be in there.

As per our [documentation](https://bikeindex.org/documentation/api_v3), time is represented by unix timestamps.

Reply:
> I don't know that you need to include this delegate at all?

Why's that?

Here's what I'm seeing:

With:

<img width="523" alt="with" src="https://user-images.githubusercontent.com/4433943/65728301-784d0b80-e088-11e9-9d47-d8ac481938d1.png">


Without:

<img width="518" alt="without" src="https://user-images.githubusercontent.com/4433943/65728307-7aaf6580-e088-11e9-8a49-2135c80c4bbf.png">

Reply:
> Also, I don't know why date_stolen_string

Ah thanks -- that was to fix the date rendering in the search listing but `convertTime` should be handling that 👍 Updated in 8b5414b8